caom2/caom2/diff.py

```diff
@@ -360,6 +360,11 @@ def _get_dict_differences(expected, actual, parent):
     """Reports on how two dictionaries are different."""
     report = []
     for expected_key, expected_value in expected.items():
+        if expected_key == 'meta_producer':
```

Comment:
Wouldn't it be more useful if this was a global config variable, e.g. `ignore_keys` list?

Reply:
Done.